src/app/pages/apps/store/available-apps-store.service.ts

```diff
@@ -0,0 +1,274 @@
+import { Injectable } from '@angular/core';
+import { UntilDestroy, untilDestroyed } from '@ngneat/until-destroy';
+import { ComponentStore } from '@ngrx/component-store';
+import { TranslateService } from '@ngx-translate/core';
+import _ from 'lodash';
+import {
+  Observable, catchError, combineLatest, map, of, switchMap, tap,
+} from 'rxjs';
+import { AppExtraCategory } from 'app/enums/app-extra-category.enum';
+import { AppsFiltersSort, AppsFiltersValues } from 'app/interfaces/apps-filters-values.interface';
+import { AvailableApp } from 'app/interfaces/available-app.interfase';
+import { WebsocketError } from 'app/interfaces/websocket-error.interface';
+import { ApplicationsService } from 'app/pages/apps/services/applications.service';
+import { DialogService } from 'app/services';
+import { ErrorHandlerService } from 'app/services/error-handler.service';
+
+export const availableAppsShownPerCategory = 6;
+
+export interface AppsByCategory {
+  title: string;
+  apps: AvailableApp[];
+  totalApps: number;
+  category: string;
+}
+
+export interface AvailableAppsState {
+  availableApps: AvailableApp[];
+  latestApps: AvailableApp[];
+  recommendedApps: AvailableApp[];
+  categories: string[];
+  appsByCategories: AppsByCategory[];
+  filteredApps: AvailableApp[];
+  filter: AppsFiltersValues;
+  isLoading: boolean;
+  isFilterApplied: boolean;
+  appsPerCategory: typeof availableAppsShownPerCategory;
```

Comment:
If the value is static, seems like an overkill to have it in state. You can just have a constant or a readonly property.